source/Flavors.rst

```diff
@@ -6,23 +6,17 @@ Flavors
 
 The STFC Cloud provides a number of flavors to users but these fall into 3 categories at the moment.
 
+No flavor has resilient storage. If this is required you should boot from Volume at VM creation time
```

Comment:
Do we have a link to how we can do this? If not, could we create an issue to go back and put this on at some point

Reply:
We dont at the moment and in fact our vm creation doc advises against it so we should add a task to do this

Reply:
I'll put together a pull request to add a note to the vm creation doc for this in the first instance